src/app/project/board/board.component.ts

```diff
@@ -262,6 +262,22 @@ export class BoardComponent implements OnInit {
         this.keywords = this.projectMetadata['keywords'];
     }
 
+    // download metadata
+    downloadMetadata() {
```

Comment:
question: will you use this method only for JSON or as a generic one for the download format?

Reply:
Once the endpoints for other formats will be available, this method will be converted into generic one where format will be passed as a parameter.

Reply:
I think there will be no separate endpoint for downloads, but just different parameter to pass to the GET method. But this needs to be confirmed with BE team.

Reply:
We should create issue for adding endpoints for different formats in backend and prioritise it.